lm_eval/models/optimum_lm_genai.py

```diff
@@ -0,0 +1,86 @@
+from importlib.util import find_spec
+from pathlib import Path
+
+from lm_eval.api.registry import register_model
+from lm_eval.models.huggingface import HFLM
+
+import sys
+for path in sys.path:
+    print(path)
+
+from llm_bench.python.llm_bench_utils.ov_utils import create_text_gen_model, build_ov_tokenizer
```

Comment:
@eaidova, please take a look as well. I think we don't need it as there is a path for OV models and LLM-Bench does not bring anything extra.